src/views/common/CourseList.vue

```diff
@@ -52,6 +53,10 @@
                 type: Boolean,
                 required: true,
             },
+            isCourseCatalogue: {
+                type: Boolean,
+                default: () => false,
```

Comment:
why is this a function?

Reply:
This was a copy and paste mistake as we used the default value for objects which types are not supported by default for props